src/api/images.tsx

```diff
@@ -79,9 +79,10 @@ export const createImageFromInstanceSnapshot = (
   instance: LxdInstance,
   snapshot: LxdInstanceSnapshot,
   isPublic: boolean,
+  project: string,
 ): Promise<LxdOperationResponse> => {
   return new Promise((resolve, reject) => {
-    fetch("/1.0/images", {
+    fetch(`/1.0/images?project=${project}`, {
```

Comment:
```suggestion
    fetch(`/1.0/images?project=${instance.project}`, {
```
We don't need to add project to the interface as the instance already contains the project info.

Reply:
aah yeah sorry blonde moment, will fix that